code/modules/library/lib_items.dm

```diff
@@ -94,6 +95,8 @@
 			else
 				choice.loc = get_turf(src)
 			update_icon()
+		if(choice == "Cancel")

```

Comment:
This has no code impact.
If you want the user to cancel, there has to be a Cancel button and this has to come before the previous code block.